recipes/pangomm/all/conanfile.py

```diff
@@ -1,101 +1,123 @@
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.meson import MesonToolchain, Meson
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
 
 
 class PangommConan(ConanFile):
     name = "pangomm"
-    homepage = "https://gitlab.gnome.org/GNOME/pangomm"
+    description = "pangomm is a C++ API for Pango: a library for layout and rendering of text."
     license = "LGPL-2.1"
     url = "https://github.com/conan-io/conan-center-index"
-    description = "pangomm is a C++ API for Pango: a library for layout and rendering of text."
+    homepage = "https://gitlab.gnome.org/GNOME/pangomm"
     topics = ["pango", "wrapper", "text rendering", "fonts", "freedesktop"]
-    settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False], "fPIC": [True, False]}
-    default_options = {"shared": False, "fPIC": True}
-
-    generators = "pkg_config"
-    exports_sources = "patches/**"
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
 
     @property
     def _is_2_48_api(self):
-        return tools.Version(self.version) >= "2.48.0"
+        return Version(self.version) >= "2.48.0"
 
     @property
     def _is_1_4_api(self):
-        return tools.Version(self.version) >= "1.4.0" and tools.Version(
-            self.version) < "2.48.0"
+        return "1.4.0" <= Version(self.version) < "2.48.0"
 
     @property
     def _api_version(self):
         return "2.48" if self._is_2_48_api else "1.4"
 
-    def validate(self):
-        if hasattr(self, "settings_build") and tools.cross_building(self):
-            raise ConanInvalidConfiguration("Cross-building not implemented")
-
-        if self.settings.compiler.get_safe("cppstd"):
-            if self._is_2_48_api:
-                tools.check_min_cppstd(self, 17)
-            elif self._is_1_4_api:
-                tools.check_min_cppstd(self, 11)
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def export_sources(self):
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
-    def build_requirements(self):
-        self.build_requires("meson/0.59.1")
-        self.build_requires("pkgconf/1.7.4")
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("pango/1.50.7")
+        self.requires("pango/1.50.14", transitive_headers=True)
```

Comment:
Where does this .14 come from? Latest available one in CCI is .10

Reply:
It's added by #18874.